static/app/views/dashboardsV2/detail.tsx

```diff
@@ -60,6 +61,20 @@ class DashboardDetail extends React.Component<Props, State> {
     window.removeEventListener('beforeunload', this.onUnload);
   }
 
+  getDashboardState() {
```

Comment:
I need to redirect to the "Create Dashboard" view from the Manage Dashboards page and the easy way was to leverage `location.state`. I don't see this pattern being used anywhere else so I'm guessing this might not be the best option. Kevan suggested I could just render <DashboardDetail/> from inside the manage dashboard view but I don't think I have the right Props in the Manage Dashboard view (specifically route, routes) that I can do that. Could that be fixed by refactoring the routes and nesting `dashboards/manage/` under `dashboards/`?

Reply:
@shruthilayaj Would the flow be easier if we have a dedicated `dashboards/new/` route for creating dashboards? This does not currently exist yet.

Reply:
I like the idea of having a new route

Reply:
Putting this back in draft while I make this change